content/beam/capability/2016/03/17/capability-matrix.html

```diff
@@ -154,19 +154,23 @@ <h1 class="post-title" itemprop="name headline">Clarifying & Formalizing Runner
     
     
 
-    <td width="25%" class="cap-summary" style="background-color:#ec3;border-color:#ca1"></td>
+    <td width="25%" class="cap-summary" style="background-color:#ec3;border-color:#ca1"><center><b>&#x2713;</b></center>
```

Comment:
I think there's a distinct possibility that we are using different versions -- and we are stomping on each others generated code, and producing a lot of updates in each PR (unnecessarily). Not sure can we do something about it -- seems unlikely.